src/pages/club/index.js

```diff
@@ -0,0 +1,55 @@
+import { Container } from '@chakra-ui/react'
+import { Box, ChakraProvider, Image } from '@chakra-ui/react'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import React from 'react'
+
+// import Masonry from 'react-masonry-css'
+import { Layout, PageTitle } from '~components'
+
+export default function Club({ header, images }) {
+  return (
+    <Layout scrollHeight={100} seo={{ header }}>
```

Comment:
```suggestion
    <Layout scrollHeight={100} seo={{ title }}>
```

We use `next-seo` package and it doesn't have `header` property, we could use `title` instead. You can see which props next-seo receives below

```ts
export interface NextSeoProps {
    title?: string;
    titleTemplate?: string;
    defaultTitle?: string;
    noindex?: boolean;
    nofollow?: boolean;
    robotsProps?: AdditionalRobotsProps;
    description?: string;
    canonical?: string;
    mobileAlternate?: MobileAlternate;
    languageAlternates?: ReadonlyArray<LanguageAlternate>;
    openGraph?: OpenGraph;
    facebook?: {
        appId: string;
    };
    twitter?: Twitter;
    additionalMetaTags?: ReadonlyArray<MetaTag>;
    additionalLinkTags?: ReadonlyArray<LinkTag>;
}
```